packages/curvenote-cli/src/session/plugins.ts

```diff
@@ -20,15 +20,15 @@ export async function loadProjectPlugins(session: ISession): Promise<CurvenotePl
   }
   // Already validated by myst plugin loader
   const modules = await Promise.all(
-    config?.plugins?.map(async (filename) => {
-      if (fs.statSync(filename).isFile() && filename.endsWith('.mjs')) {
+    config?.plugins?.map(async (plugin) => {
+      if (fs.statSync(plugin.path).isFile() && plugin.path.endsWith('.mjs')) {
```

Comment:
Ah makes sense - these changes are required due to the update to `plugins` here https://github.com/jupyter-book/mystmd/pull/1329/files#diff-9a016dcc5b42d8eb84f7d9e7caa57d2f23ef43e374e635b9cf2e333ce068f2f9R20